src/Integration.Vsix/Suppression/SuppressionManager.cs

```diff
@@ -107,7 +107,7 @@ private bool ShouldIssueBeReported(Diagnostic diagnostic)
             bool matchFound = issuesInFile.Any(i =>
                     liveIssue.StartLine == i.Line ||
                     StringComparer.Ordinal.Equals(liveIssue.LineHash, i.Hash));
-            return matchFound;
+            return !matchFound;
```

Comment:
Oops.
The method is called _ShouldIssueBeReported_ so we should not report if the live issue matches a suppressed issue on the server. I'll add unit tests shortly (this + 1 other commit to fix the connection issues and the end to end will be working).